src/core/include/cesium/omniverse/FabricMaterialDefinition.h

```diff
@@ -17,14 +19,20 @@ class FabricMaterialDefinition {
         const CesiumGltf::MeshPrimitive& primitive,
         const MaterialInfo& materialInfo,
         const FeaturesInfo& featuresInfo,
-        uint64_t imageryLayerCount,
+        const ImageryLayersInfo& imageryLayersInfo,
         bool disableTextures,
         const pxr::SdfPath& tilesetMaterialPath);
 
     [[nodiscard]] bool hasVertexColors() const;
     [[nodiscard]] bool hasBaseColorTexture() const;
     [[nodiscard]] const std::vector<FeatureIdType>& getFeatureIdTypes() const;
+    [[nodiscard]] const std::vector<OverlayType>& getImageryOverlayTypes() const;
+    [[nodiscard]] const std::vector<OverlayRenderPipe>& getImageryOverlayRenderPipes() const;
     [[nodiscard]] uint64_t getImageryLayerCount() const;
+    [[nodiscard]] uint64_t getPolygonImageryCount() const;
+    [[nodiscard]] uint64_t getIonImageryCount() const;
+    [[nodiscard]] std::vector<int> getIonImageryLayerIndices() const;
+    [[nodiscard]] std::vector<int> getPolygonImageryLayerIndices() const;
```

Comment:
I think these four getters can be removed

Reply:
Good call.

I think we can get Visual Studio Code to detect methods with no references. It would be very helpful, at least for me. I created an issue to look into it.